src/nrnpython/nrnpython.h

```diff
@@ -17,10 +17,6 @@
 #if defined(USE_PYTHON)
 #undef _POSIX_C_SOURCE
 #undef _XOPEN_SOURCE
-#if defined(__MINGW32__)
-//at least a problem with g++6.3.0
-#define _hypot hypot
```

Comment:
Just wondering if it is related to this PR ?

Reply:
I dropped this here because the same logic is there in `nrnwrap_Python.h`:
https://github.com/neuronsimulator/nrn/blob/422d15fc83ac9dd433836928fde3da5cc6886546/src/nrnpython/nrnwrap_Python.h#L5-L8
which we include just below.

Reply:
Right